source/clientsideapp.md

```diff
@@ -105,7 +105,69 @@ Also, an `index.html` file needs to be at the root of your repository with your
 
 ### Deploying the application
 
-Once we've packaged our application and did a `Hello World!` in our `index.html`, we must make it available for installation. To achieve this, we can create a [Github](https://github.com/) repository, and push our work on it.
+Once we've packaged our application and did a `Hello World!` in our `index.html`, we must make it available for installation.
+
+Thanks to our awesome team members, you've got not only one nor two, but three solutions to deploy your application in your local environment. Let's see what you can use!
+
+#### 1. Serve local files throught a proxy inside the VM
+
+This solution is dedicated to client-side only apps, and is the prefered way to test your app.
+
+In your terminal, simply go to your app directory, and run the `cozy-dev proxify` command.
+
+
+```
+cd myapp
+cozy-dev proxify http://localhost:9104
+```
+
+
+This command uses up to two params that can let you customize the proxy:
+
+1. a URL: we here use the local VM (instanciated with `cozy-dev vm:start`), but you can safely use any cozy available remotely. _No more need to use a local VM for development_, you can safely use your own Cozy (like `tbl.cozycloud.cc`)
```

Comment:
Why not using title instead? 
Adding line breaks before your note: "_No more need to use a local ..." ?